INTEROP.md

```diff
@@ -337,6 +337,22 @@ manually:
 
 In all cases the C string is supposed to be encoded as UTF-8.
 
+### Scope-local pointers ###
+
+It is possible to create scope-stable pointer of C representation of `CValues<T>`
+instance using `MemScope.ptr` extension property. It allows to use APIs which requires C
```

Comment:
It's better to refer this property as "`CValues<T>.ptr` extension property available under `memScoped { ... }`".

Reply:
Done